pkg/controller/machine/controller.go

```diff
@@ -211,6 +211,7 @@ func (r *ReconcileMachine) Reconcile(request reconcile.Request) (reconcile.Resul
 				return reconcile.Result{Requeue: true, RequeueAfter: requeueErr.RequeueAfter}, nil
 			}
 
+			klog.Errorf("Error updating machine %q: %v", name, err)
```

Comment:
```suggestion
			klog.Errorf("Error updating machine "%s/%s": %v", m.Namespace, name, err)
```

Not related to this change, but I we need to update other logging to use namespace/name instead of just name as well.

Reply:
accepted, subbed in some backticks